releasenotes/notes/call_extract_once_per_request-620e1680c6fa6a85.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an issue where Django, Flask, Cherrypy, wsgi, 
+    asgi, pyramid, molten, falcon, tornado, aiohttp, bottle, rq, and sanic integrations
+    were activating distributed tracing headers multiple times on every request. This is 
```

Comment:
Can we elaborate on the impact of this fix? Does it improve performance, fix span parenting, etc.